vms/avm/block/executor/block_test.go

```diff
@@ -45,6 +53,9 @@ func TestBlockVerify(t *testing.T) {
 				b := &Block{
 					Block: mockBlock,
 					manager: &manager{
+						backend: &executor.Backend{
```

Comment:
we introduce the `backend`s here since we plan to handle dynamic fees via a `fee.Calculator` struct that will require `Config` to be duly set.
Setting `backend`s here reduces diffs down the line

Reply:
There are some minor variations. I consolidated backends into a little helper